src/test/lambda/local/debugConfiguration.test.ts

```diff
@@ -91,30 +91,12 @@ describe.only('makeCoreCLRDebugConfiguration', async () => {
 
             assert.strictEqual(config.windows.pipeTransport.pipeProgram, 'powershell')
         })
-
-        it('uses the specified port', async () => {
-            const config = await makeConfig({ port: 538 })
-
-            assert.strictEqual(
-                config.windows.pipeTransport.pipeArgs.some(arg => arg.includes('538')),
-                true
-            )
-        })
```

Comment:
these tests are meanginless even if we were still using this parameter (but it was removed anyway)